tests/operator/operator.go

```diff
@@ -3285,3 +3254,25 @@ func getVirtLauncherSha(deploymentConfigStr string) (string, error) {
 
 	return config.VirtLauncherSha, nil
 }
+
+func deleteAllKvAndWait(virtClient kubecli.KubevirtClient, ignoreOriginal bool, originalKvName string) {
```

Comment:
There's no much point in passing `virtClient` around.
Setting `virtClient := kubevirt.Client()` here would be less disturbing the the users of this function.